gcpy/benchmark/config/1mo_benchmark.yml

```diff
@@ -47,13 +47,17 @@ data:
       dir: GCC_ref
       outputs_subdir: OutputDir
       restarts_subdir: Restarts
+      logs_subdir: .
+      logs_template: "log.%Y%m%d"
```

Comment:
This seems to assume a specific log file name (which obviously the user can edit). Has the analogous change to the log file name been made to the sample run scripts in the geos-chem repository?

Reply:
Thanks @msulprizio.  I could name the log file here `GC.log` to correspond to the 1-month benchmark default file name.  This file isn't used as much since the 1-month benchmarks are done on the cloud (but sometimes we might need to run one manually).